packages/pricing/src/models/price-set-money-amount.ts

```diff
@@ -30,6 +36,28 @@ export default class PriceSetMoneyAmount {
   })
   money_amount?: MoneyAmount
 
+  @Property({ columnType: "numeric", default: 0 })
+  number_rules?: string
```

Comment:
todo: type issue `number` also, should we remove the optional here since there is always a default value and add it instead in the symbol optional props?

Reply:
I believe it is safe to set this column `number_rules` as Integer. Same on the migration.

Reply:
Since we are using a `numeric` type it's saved as a string in the database and mikro-orm doesn't automatically parse values and returns strings (you can see this in the integration tests `calculate-price.spec.ts`)

We can make this a number by adding `serializer: Number` to the options of the `Property`.

Reply:
I think we should have the serializer as we have seen in another pr if I recal, wdyt @carlos-r-l-rodrigues

Reply:
To me, this field is an Integer. So just changing the type should work to serialize it correctly.